devicon.json

```
@@ -2455,6 +2455,27 @@
         "color": "#2c2255",
         "aliases": []
     },
+    {
+        "name": "ecto",
+        "altnames": [],
+        "tags": [
+            "data",
+            "elixir",
+            "integration",
+            "query"
+        ],
+        "versions": {
+            "svg": [
+                "original",
+                "plain"
+            ],
+            "font": [
+                "plain"
+            ]
+        },
+        "color": "#77bf43",
+        "aliases": []
```

Since `original` is an icon with just one solid color, we can make an alias to `plain` version and remove the current `plain` version.
```suggestion
        "versions": {
            "svg": [
                "original"
            ],
            "font": [
                "original"
            ]
        },
        "color": "#77bf43",
        "aliases": [
            {
                "base": "original",
                "alias": "plain"
            }
        ]
```